package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "next dev -p 3000",
     "lint": "eslint --ext .ts,.tsx '.'",
     "start": "next start",
-    "test": "jest"
+    "test": "npm run build && jest"
```

Comment:
Adding build to test script to catch build errors during the jenkins test stage. Otherwise, these errors will only show up during the build which happens after pushing to develop.